Makefile-Android

```diff
@@ -44,7 +44,7 @@ ENABLE_TARGETED_LISTEN          ?= 0
 ANDROID_RELEASE                 ?= 22
 
 # Deprecated -- Use ANDROID_STL
```

Comment:
Does this comment need to change to reflect the commit / description for this PR?

Reply:
I've looked at the Makefile, and the fix is correct, and comment may remain unchanged.  The "deprecated" annotation refers to the use of the variable: the variable was deprecated (should not be used to modify behavior by external programs) but is still used as a base value for `ANDROID_STL`.  Given the passage of time, I would be satisfied if the `ANDROID_CXX` was removed altogether, and the default value of `ANDROID_STL` were changed to `libc++`

Reply:
FWIW, I'm testing the internal build with the above suggestion implemented; if nothing relies on `ANDROID_CXX` we should remove the deprecated variable

Reply:
Yep, I have updated PR and removed ANDROID_CXX.